build-config/resources/Ballerina.toml

```
@@ -1,12 +1,13 @@
 [package]
 org = "ballerina"
 name = "data.jsondata"
-version = "@toml.version@"
+version = "0.1.0"
```

```suggestion
version = "@toml.version@"
```
This is wrong. If we keep as it will use the version mentioned in the `gradle.properties`

---

+1